src/core/modules/GridLinesMgr.cpp

```diff
@@ -1756,7 +1777,35 @@ void SkyPoint::draw(StelCore *core) const
 				sPainter.drawText(-dir, southernLabel, 0, shift, shift, false);
 			}
 			break;
-		}		
+		}
+		case LAGRANGE_SOLAR:
+		{
+			// Takes the positional vector of the planet from the solar object
+			Vec3d pos= core->getCurrentObserver()->getHomePlanet()->getHeliocentricEclipticPos();
+			// Creates points by adding/subtracting 60 degrees to the vector along the ecliptic plane
```

Comment:
This approach may work for circular orbits. How does it appear for e.g. Mars or Mercury? Its distance from the sun will place it far off the orbit distance at that point.

Reply:
See commit [78d81e7](https://github.com/Stellarium/stellarium/pull/3090/commits/78d81e7851306c086473ab1541b39cb84eb1b214) 
I am thinking it should account for deviations from orbit, but I am not really an astronomer.